internal/rpmmd/repository_test.go

```diff
@@ -0,0 +1,48 @@
+package rpmmd
```

Comment:
Very nice! Just as mentioned in the chat, I'd make this a part of the integration test, by creating a new binary that is shipped in the -test rpm subpackage.